config/software/chefdk.rb

```diff
@@ -56,9 +56,9 @@
 
   # Perform multiple gem installs to better isolate/debug failures
   {
-    'chefspec'          => '4.2.0.beta.1',
+    'chefspec'          => '4.2.0',
     'fauxhai'           => '2.2.0',
-    'rubocop'           => '0.18.1',
+    'rubocop'           => '0.28.0',
     'knife-spork'       => '1.4.2',
```

Comment:
I think the right thing to do is follow the same general plan I outlined in https://github.com/chef/chef-dk/issues/282 That is, we should have a secondary Gemfile with plugins and use that to generate the Gemfile.lock for the build and the appbundler executable. This will lock the versions of plugins and their deps to known working versions at build time.